### ISSUE TITLE
"Benutzer" und "Benutzergruppen" werden in das Feld "Aktuell angemeldeter Nutzer" aufgenommen

### ISSUE THREAD

**Issue by FrodBRA** [SVWS-NRW/SVWS-Server#246]
Entsprechend mit Berechtigungen.

**Comment by hmt** [comment]
Das ist ungewohnt. Diesen Bereich sehen eigentlich nur die Admins, sollte im alltag überhaupt mit dem Admin gearbeitet werden? Ich käme nie auf die Idee, Benutzer/gruppen unter meinem eigenen Account zu suchen.

**Comment by FrodBRA** [comment]
Wenn du als ein Admin-Nutzer angemeldet bist, hat der dort auch Admin-Berechtigungen.

Wenn du nicht als ein Admin-Nutzer angemeldet bist, hat der dort natürlich keine Admin-Berechtigungen.

**Comment by hmt** [comment]
doch, schon, aber da macht man doch die Einstellungen, die den eigenen Nutzer betreffen und nicht die der anderen?

**Comment by FrodBRA** [comment]
Das Argument, dass das natürlich keine Nutzereinstellungen sind, finde ich stichhaltig.

Zuerst hatten wir diskutiert, Benutzer/Benutzergruppen unter einem Punkt zusammenzufassen bzw. als Unterordner anzulegen. Das würde dann mehr Sinn machen als die Aufnahme in den "Aktuellen Benutzer"?

**Comment by hmt** [comment]
Ja, ich würde das als Benutzerverwaltung in der Schule lassen

**Comment by ThomasBachran** [comment]
Oder wir tauschen die App `Ansicht` gegen eine App `Einstellungen` aus, wo dann zusätzlich auch die Benutzerverwaltung zu finden ist. Was meint ihr?

![grafik](https://github.com/user-attachments/assets/dff3b779-1ba2-484d-bf2a-4692406b1cc9)

**Comment by hmt** [comment]
Die Ansichtssachen können in den Benutzer rein, stattdessen unten Administration? So wie GitLab das macht? Dann kommt diese App auch nur, wenn man Admin ist. Hat den Vorteil, dass man sieht, wenn man Admin ist im Vergleich zu anderen Nutzern und es ist nicht im Weg
